src/main/java/Duke.java

```diff
@@ -1,10 +1,15 @@
+import java.io.IOException;
+
 public class Duke {
+    /**
+     * Main driver of the program.
+     * @param args Commandline arguments, not used
```

Comment:
The coding style guide specifies punctuation at the end of each javadoc line (so I think you have to add a full stop for all your parameter descriptions in the other files as well)